src/main/java/seedu/address/logic/commands/mode/SwitchCommand.java

```diff
@@ -12,12 +12,14 @@
 
     public static final String COMMAND_WORD = "switch";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Switches mode. Parameters: "
-            + ModeEnum.getModeOptions()
-            + " \n"
-            + "Example: " + COMMAND_WORD + " contact";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Switches mode.\n"
+            + "Parameters: MODE (must be " + ModeEnum.getModeOptions()
+            + ")\n"
+            + "Example: " + COMMAND_WORD + " " + ModeEnum.DELIVERABLE.getArgument();
+
 
     public static final String MESSAGE_SUCCESS = "Mode switched to: %1$s";
+    public static final String MESSAGE_SAME_MODE = "Switched to same mode: %s!";
```

Comment:
Consider using `You are already in this mode!` or `Mode is presently displayed!` so it appears more like the user made a mistake.